### PR TITLE
Some documentation rewriting/formatting

### PR DESCRIPTION
**Pull request by apenney**
This chart is great but the documentation suffered from some typos and clumsy English.  I think it was an ESL contributor, and they certainly write better English than I can write their language, so I hope nobody will feel like this is a criticism of a fantastic chart!

### CODE REVIEW

File: incubator/cassandra/README.md [hunk 1]
```diff
@@ -1,60 +1,84 @@
...
-To install the Cassandra Chart into your Kubernetes cluster (This Chart requires persistent volume by default, you may need to create a storage class before install chart. To create storage class, see [Persist data](#persist_data) section)
+Cassandra is a distributed NoSQL database management system designed to handle
+large amounts of data across many commodity servers, providing high
+availability with no single point of failure.
+
+## Installing Chart
+ 
+Installation of this Chart requires PV (Persistent Volume) storage.  You may
+need to create a storage class before you install.  To create this, see the
+[Persist Data](#persisting_data) section.
```
Comment (sameersbn) [inline incubator/cassandra/README.md]: `[Persist Data](#persisting_data)` -> `[Persisting Data](#persisting-data)`

File: incubator/cassandra/README.md [hunk 2]
```diff
@@ -1,60 +1,84 @@
 # Cassandra
-A Cassandra Chart for Kubernetes
 
-## Install Chart
-To install the Cassandra Chart into your Kubernetes cluster (This Chart requires persistent volume by default, you may need to create a storage class before install chart. To create storage class, see [Persist data](#persist_data) section)
+Cassandra is a distributed NoSQL database management system designed to handle
```
Comment (sameersbn) [inline incubator/cassandra/README.md:3]: why the link breaks? I prefer the markdown rendering to handle wrapping the text according to the layout

Comment (apenney) [reply incubator/cassandra/README.md:3]: I just find it easier to wrap with 80 char linebreaks, same rule most people use for code, if it really bothers people I can totally go the opposite way and let the editor soft break.

File: incubator/cassandra/README.md (hunk 1 above)
Comment (apenney) [reply incubator/cassandra/README.md]: nice catch, fixed!

File: incubator/cassandra/README.md (hunk 2 above)
Comment (dhilipkumars) [reply incubator/cassandra/README.md:3]: I agree.  Most of the docs in kubernetes project is 80 (or slightly higher) wrapped.  Is easier to maintian than unwrapped.

File: incubator/cassandra/README.md [hunk 3]
```diff
@@ -1,60 +1,84 @@
...
   enabled: true
 ```
 
-If you want to create a `StorageClass` on other platform, please see documentation here [https://kubernetes.io/docs/user-guide/persistent-volumes/](https://kubernetes.io/docs/user-guide/persistent-volumes/)
+If you want to create a `StorageClass` on other platform, please see
+documentation here
+[https://kubernetes.io/docs/user-guide/persistent-volumes/](https://kubernetes.io/docs/user-guide/persistent-volumes/)
+
 
+## Sizing Cassandra Cluster
```
Comment (dhilipkumars) [inline incubator/cassandra/README.md:52]: `"## Install Chart with specific cluster size"` this the older version is much more understandable in my opinion.

File: incubator/cassandra/README.md [hunk 4]
```diff
@@ -1,60 +1,84 @@
...
 
 ```bash
 helm install --namespace "cassandra" -n "cassandra" --set config.cluster_size=5 incubator/cassandra/
 ```
 
-## Install Chart with specific resource size
-By default, this Chart will create a cassandra with CPU 2 vCPU and 4Gi of memery which is suitable for development environment.
-If you want to use this Chart for production, I would recommend to update the CPU to 4 vCPU and 16Gi. Also increase size of `max_heap_size` and `heap_new_size`.
-To update the settings, edit `values.yaml`
+## Modifying resource limits
```
Comment (dhilipkumars) [inline incubator/cassandra/README.md:64]: Same here i would either go with the older heading 
`## Install Chart with specific resource size`
or say 
`Modifying the default resource limits`

**Comment by dhilipkumars** [comment]
Marking as stale.

**Comment by unguiculus** [comment]
Closing as stale. Feel free to reopen if/when you decide to work on this again.

**Pull request closed without merging**